src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc

```diff
@@ -0,0 +1,187 @@
+= Hawkular Alerts with OpenTracing
+John Mazzitelli
+2017-09-06
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, opentracing, jaeger
+:figure-caption!:
+
+Two recent blogs discuss how OpenTracing instrumentation can be used to collect application metrics:
+
+* http://www.hawkular.org/blog/2017/06/26/opentracing-appmetrics.html
+* http://www.hawkular.org/blog/2017/08/opentracing-appmetrics-canary.html
+
+A further interesting integration can be the addition of Hawkular Alerts to the environment.
+
+As the previous http://www.hawkular.org/blog/2017/08/alerts-multiple-sources.html[blog] and https://www.youtube.com/watch?v=mM1mwJneKO4[demo] discuss, Hawkular Alerts is a generic, federated alerts system that can trigger events, alerts, and notifications from different, independent systems such as Prometheus, ElasticSearch, and Kafka.
+
+Here we can combine the two. Let's follow the directions for the OpenTracing demo (using the Jaeger implementation) and add Hawkular Alerts.
+
+What this can show is OpenTracing application metrics triggering alerts when (as in this example) OpenTracing spans encounter a larger-than-expected error rates.
+
+(Note: these instructions assume you are using Kubernetes / Minikube - see the Hawkular OpenTracing blogs linked above for more details on these instructions)
+
+=== START KUBERNETES
+
+Here we start minikube giving it enough resources to run all of the pods necessary for this demo. We also start up a browser pointing to the Kubernetes dashboard, so you can follow the progress of the remaining instructions.
+
+```
+minikube start --cpus 4 --memory 8192
+
+minikube dashboard
+```
+
+=== DEPLOY PROMETHEUS
+
+```
+kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
```

Comment:
Yesterday I had to update the example instructions to base it on prom-operation v0.11.0, to get it working consistently on kubernetes and openshift, so may be best if you update this to be:
`kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.11.0/bundle.yaml`.

I will need to update the other blog posts to make sure they are consistent. So this now just uses the v1alpha1.

Reply:
see latest commit that makes this change.

Reply:
Thanks 👍